.github/workflows/build-and-push-dev-images.yml

```diff
@@ -78,30 +74,14 @@ jobs:
               echo "PUSH: Rebuilding changed images only"
```

Comment:
There is still a `compat=0` / `compat=1` we can remove a few lines above this one